crates/rooch/src/commands/move_cli/commands/explain.rs

```diff
@@ -55,18 +57,25 @@ impl Explain {
         };
 
         match error_description_bytes {
-            Some(bytes) => match get_explanation(&module_id, self.abort_code, bytes) {
-                None => println!(
-                    "Unable to find a description for {}::{}",
-                    self.location, self.abort_code
-                ),
-                Some(error_desc) => println!(
-                    "Category:\n  Name: {}\n  Description: {}",
-                    error_desc.code_name, error_desc.code_description,
-                ),
-            },
+            Some(bytes) => {
+                let explain_result =
+                    explain_move_abort(AbortLocation::Module(module_id), self.abort_code, bytes);
+                match explain_result {
+                    None => {
+                        return Err(anyhow::Error::msg(format!(
+                            "Unable to find a description for {}::{}",
+                            self.location, self.abort_code
+                        )));
+                    }
+                    Some(error_desc) => println!(
+                        "Category:\n  Name: {}\n  Description: {}",
+                        error_desc.reason_name.unwrap(),
+                        error_desc.code_description.unwrap(),
```

Comment:
the description can be None

Reply:
"description" is the comment for the abort code, but it seems like something is missing.